src/environmentd/src/http.rs

```diff
@@ -439,9 +439,22 @@ impl InternalHttpServer {
                 routing::get(console::handle_internal_console),
             )
             .layer(middleware::from_fn(internal_http_auth))
-            .layer(Extension(adapter_client_rx.shared()))
+            .layer(Extension(adapter_client_rx_shared.clone()))
             .layer(Extension(console_config))
-            .layer(Extension(active_connection_count));
+            .layer(Extension(Arc::clone(&active_connection_count)));
+
+        let ws_router = Router::new()
+            .route("/api/experimental/sql", routing::get(sql::handle_sql_ws))
```

Comment:
But we didn't do that for the /api/sql URL! IMO this is morally equivalent to the public API and so it makes sense to use  the same path as the public API to indicate the correspondence. Also it's one less conditional in the console codebase.

Reply:
Yep I left this the same so that there would be no changes needed on the web console - it can just use the same code without a conditional (and that's how I was able to test that it worked)